toolkit/recordreplay/ProcessRecordReplay.cpp

```diff
@@ -722,7 +722,7 @@ static bool FilterMatches(const InfallibleVector<JSFilter>& aFilters,
 }
 
 const char* CurrentFirefoxVersion() {
-  return "74.0a1";
+  return "86.0";
```

Comment:
Definitely a question for Brian since I have no idea why we hardcode this in the first place. Seems reasonable though since this is what's in `gecko-dev/browser/config/version.txt`.

Reply:
This should match the browser version we merged from upstream, it looks like I forgot to fix this in the last merge.  It would be nice to either add instructions to update this to the "Merging from upstream" section of README.md, or (more work) restructure this to avoid needing to be baked in.